src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -31,8 +31,9 @@ import { DivisionSchemes } from '../../../../../data/constants';
 const courseId = 'course-v1:edX+TestX+Test_Course';
 const defaultAppConfig = {
   id: 'legacy',
-  divideByCohorts: false,
-  divideCourseTopicsByCohorts: false,
+  divideByCohorts: true,
+  divideCourseTopicsByCohorts: true,
+  alwaysDivideInlineDiscussions: true,
```

Comment:
are not these default configurations need to be set as false?

Reply:
it's like when we fetch configurations for a course the defaults are false right?

Reply:
Yes. Default values are false. I'll update it